offchain-modules/packages/x/src/config.ts

```diff
@@ -163,6 +163,14 @@ export interface collectorConfig {
   gasPriceGweiLimit: number;
 }
 
+export interface monitorConfig {
+  expiredTime: number;
+  expiredCheckInterval: number;
+  discordWebHook: string;
+  scanStep: number;
+  env: string;
+}
```

Comment:
I recommend to use a separate config for monitor.
It's too heavy to include the whole ForceBridgeCore to just use it's config.

Reply:
separate config also have many config item，such as ckb deps group，white list，and multisigLockscript，ownerCellTypescript。It won't too much less then ForceBridgeConfig